dbt/macros/translate_tables.sql

```diff
@@ -0,0 +1,23 @@
+{% macro translate_motif(field) -%}
```

Comment:
```suggestion
{% macro translate_motif_refus(field) -%}
```
"motif" est trop large, ça pourrais s'appliquer à d'autres champs (présents ou futurs)